dev/breeze/doc/09_release_management_tasks.rst

```diff
@@ -689,3 +689,27 @@ default is to build ``both`` type of packages ``sdist`` and ``wheel``.
   :target: https://raw.githubusercontent.com/apache/airflow/main/dev/breeze/doc/images/output_release-management_prepare-task-sdk-distributions.svg
   :width: 100%
   :alt: Breeze release-management prepare-task-sdk-distributions
+
+
+Preparing airflow ctl packages
```

Comment:
```suggestion
Preparing airflow ctl distributions
```

:D

Reply:
And below (likely there are a few remnants in other places in the docs as well)

Reply:
Nice one :D Yes, I will update all of them :)

Reply:
Updated all packages to distributions in the document